src/types/ObjectType.ts

```diff
@@ -28,7 +28,8 @@ export class ObjectType extends BscType {
         return this.toString();
     }
 
-    getMemberType(name: string) {
-        return super.getMemberType(name) ?? new ReferenceType(name, () => this.memberTable);
+    getMemberType(name: string, flags: SymbolTypeFlags) {
+        // An object can only have runtime members
+        return super.getMemberType(name, flags) ?? DynamicType.instance;
```

Comment:
Add a TODO: how do we handle accessing properties of an object (eg. something returned from `createObject()`)